Firmware/xyzcal.cpp

```diff
@@ -135,10 +135,10 @@ pos_mm_t pos_2_mm(float pos){
 	return pos * 0.01f;
 }
 
-void xyzcal_meassure_enter(void)
+void xyzcal_meassure_center(void)
```

Comment:
I think this is fixing the wrong typo. We had enter -> leave. Now "center" doesn't make much sense.
I would instead have fixed "meassure" to "measure"